gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -25,6 +25,48 @@ local MutedState
 
 g_VoicePanelList = nil
 
+local function VoiceTryEnable()
+	if not hook.Run("TTT2CanUseVoiceChat", LocalPlayer()) then return end
+
+	if not VOICE.IsSpeaking() and VOICE.CanSpeak() then
+		VOICE.IsTeam = false
+		RunConsoleCommand("+voicerecord")
```

Comment:
Maybe add a return for success? Because this is a try-func.

Reply:
Done